haskell_scotty/Server.hs

```diff
@@ -22,4 +22,9 @@ main = scotty 6969 $ do
     jsonAPI client
     status status201
 
+  delete "/api/clients/:id" $ do
+    clientId <- param "id"
+    liftIO $ deleteClient clientId
```

Comment:
Just guessing, but would desugaring the `do` syntax provide clarity? Is the return value actually being used, in reality, since we're in monad-land?